databricks_cli/stack/api.py

```diff
@@ -321,6 +382,39 @@ def _deploy_workspace(self, resource_properties, physical_id, overwrite):
 
         return new_physical_id, deploy_output
 
+    def _download_workspace(self, resource_properties, overwrite):
+        """
+        Download workspace asset.
+
+        :param resource_properties: dict of properties for the workspace asset. Must contain the
+        'source_path' and 'path' fields. The other fields will be inferred if not provided.
+        :param overwrite: Whether or not to overwrite the contents of workspace notebooks.
+        """
+        # Required fields. TODO(alinxie) put in _validate_config
+        local_path = resource_properties.get('source_path')
+        workspace_path = resource_properties.get('path')
+        object_type = resource_properties.get('object_type')
+        click.echo('Downloading {} from Databricks path {} to {}'.format(object_type,
+                                                                         workspace_path,
+                                                                         local_path))
+        if object_type == 'NOTEBOOK':
```

Comment:
Put this in a constant please!

Reply:
will use the constant from workspace API